src/ir_Mitsubishi.h

```diff
@@ -40,28 +40,59 @@
 #include "IRsend_test.h"
 #endif
 
+/// Native representation of a Mitsubishi 144-bit A/C message.
+union Mitsubishi144Protocol{
+  uint8_t raw[kMitsubishiACStateLength];  ///< The state in code form.
+  struct {
+    // Byte 0~4
+    uint8_t pad0[5];
+    // Byte 5
+    uint8_t       :5;
+    uint8_t Power :1;
+    uint8_t       :2;
+    // Byte 6
+    uint8_t       :3;
+    uint8_t Mode  :3;
+    uint8_t       :2;
+    // Byte 7
+    uint8_t Temp  :8;
+    // Byte 8
+    uint8_t         :4;
+    uint8_t WideVane:4;  // SwingH
+    // Byte 9
+    uint8_t Fan     :3;
+    uint8_t Vane    :3;  // SwingV
+    uint8_t VaneBit :1;
+    uint8_t FanAuto :1;
+    // Byte 10
+    uint8_t Clock   :8;
+    // Byte 11
+    uint8_t StopClock :8;
+    // Byte 12
+    uint8_t StartClock:8;
+    // Byte 13
+    uint8_t Timer :3;
+    uint8_t       :5;
+    // Byte 14~16
+    uint8_t pad1[3];
+    // Byte 17
+    uint8_t Sum   :8;
+  };
+};
 
 // Constants
-const uint8_t kMitsubishiAcModeOffset = 3;
 const uint8_t kMitsubishiAcAuto = 0b100;
 const uint8_t kMitsubishiAcCool = 0b011;
 const uint8_t kMitsubishiAcDry =  0b010;
 const uint8_t kMitsubishiAcHeat = 0b001;
-const uint8_t kMitsubishiAcPowerOffset = 5;
-const uint8_t kMitsubishiAcPower = 1 << kMitsubishiAcPowerOffset;  // 0x20
-const uint8_t kMitsubishiAcFanOffset = 0;
-const uint8_t kMitsubishiAcFanSize = 3;  // Mask 0b111
+const uint8_t kMitsubishiAcPower = 0x20;
```

Comment:
This looks like it's unused. Please remove it if that is correct.
If so, the line:
```
#define MITSUBISHI_AC_POWER kMitsubishiAcPower
```
can also be removed.